commit 6c2d65452bd5fe62723988a1a570789921900d59
Author: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>
Date:   Fri Sep 30 15:39:47 2016 +0200

    address gcc6 build error
    
    With gcc6, compiling fails with `stdlib.h: No such file or directory`,
    as including '-isystem /usr/include' breaks with gcc6, cf.,
    https://gcc.gnu.org/bugzilla/show_bug.cgi?id=70129.
    
    This commit addresses this issue for this package in the same way
    it was addressed in various other ROS packages. A list of related
    commits and pull requests is at:
    
      https://github.com/ros/rosdistro/issues/12783
    
    Signed-off-by: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>

diff --git a/stereo_image_proc/CMakeLists.txt b/stereo_image_proc/CMakeLists.txt
index b7127d2..e201436 100644
--- a/stereo_image_proc/CMakeLists.txt
+++ b/stereo_image_proc/CMakeLists.txt
@@ -16,9 +16,7 @@ catkin_package(
 include_directories(include)
 
 find_package(OpenCV REQUIRED)
-include_directories(SYSTEM ${catkin_INCLUDE_DIRS}
-                           ${OpenCV_INCLUDE_DIRS}
-)
+include_directories(${catkin_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS})
 
 # Nodelet library
 add_library(${PROJECT_NAME} src/libstereo_image_proc/processor.cpp src/nodelets/disparity.cpp src/nodelets/point_cloud2.cpp)
